garrysmod/gamemodes/sandbox/gamemode/spawnmenu/controlpanel.lua

```diff
@@ -63,7 +63,7 @@ function PANEL:ToolPresets( group, cvarlist )
 
 	preset:SetPreset( group )
 	preset:AddOption( "#preset.default", cvarlist )
-	for key, val in pairs( table.GetKeys( cvarlist ) ) do
+	for key, val in ipairs( table.GetKeys( cvarlist ) ) do

```

Comment:
Why does this even use `table.GetKeys`?

Wouldn't this be better?

```lua
for cvar, defVal in pairs( cvarlist ) do
preset:AddConVar( cvar )
end
```

Reply:
The idea is to use https://wiki.facepunch.com/gmod/Tool:BuildConVarList but I guess there is no point if they are ordered or not.

Reply:
GetKeys doesn't make them ordered either

Reply:
Yep, whatever comes first becomes 1... :D Fixed in https://github.com/Facepunch/garrysmod/pull/1854/commits/e57c7cbf70399d62161f5e185aabc2397ee693d2

Reply:
https://github.com/Facepunch/garrysmod/blob/master/garrysmod/gamemodes/sandbox/entities/weapons/gmod_tool/stool.lua#L67